logger.js

```diff
@@ -82,45 +105,41 @@ Logger.prototype.destroy = function destroy() {
     }, this);
 };
 
-module.exports = Logger;
-
-function logMethod(logger, levelName, level, path) {
-    var streams = logger.streams;
-    var logStreams = level.backends.reduce(function (acc, name) {
-        if (!streams[name]) {
-            return acc;
-        }
-
-        acc[name] = streams[name];
-        return acc;
-    }, {});
-
-    return log;
+Logger.prototype.writeEntry = function writeEntry(entry, callback) {
+    var levelName = entry.level;
+    var level = this.levels[levelName];
+    var logStreams = this._streamsByLevel[levelName];
+    var logger = this;
+    if (this.statsd && typeof this.statsd.increment === 'function') {
```

Comment:
nitpick. we probably want to put this in a transform in the future. I know you didn't put it here :)